File: businesslib/src/main/java/com/wujia/businesslib/data/MessageBean.java
package com.wujia.businesslib.data;

import com.litesuits.orm.db.annotation.PrimaryKey;
import com.litesuits.orm.db.annotation.Table;
import com.litesuits.orm.db.enums.AssignType;
import com.wujia.businesslib.data.RootResponse;

/**
 * author ：shenbingkai@163.com
 * date ：2019-03-30
 * description ：
 */
public class MessageBean extends RootResponse {

    public Message content;

    public static class Message {

        public PropertyInfo propertyInformation;
        public PropertyMessage propertyMessage;
    }

    public static class PropertyInfo {

        public String nickName;
        public String headImage;
        public String mobile;
        public String accid;
        public String communityName;
        public String communityId;
    }

    public static class PropertyMessage {

        public String title;
        public String pureText;
        public String type;
        public String id;
        public String communityId;
        public String createDate;
        public String typeMsg;
        public String senderAccId;
    }
}

File: businesslib/src/main/java/com/wujia/businesslib/data/MessageBean.kt
package com.wujia.businesslib.data

import com.wujia.lib_common.base.RootResponse

/**
 * author ：shenbingkai@163.com
 * date ：2019-03-30
 * description ：
 */
class MessageBean : RootResponse() {

    var content: Message? = null
    var _type = ""

    class Message {

        var propertyInformation: PropertyInfo? = null
        var propertyMessage: PropertyMessage? = null
    }

    class PropertyInfo {

        var nickName: String? = null
        var headImage: String? = null
        var mobile: String? = null
        var accid: String? = null
        var communityName: String? = null
        var communityId: String? = null
    }

    class PropertyMessage {

        var title: String? = null
        var pureText: String? = null
        var type: String? = null
        var id: String? = null
        var communityId: String? = null
        var createDate: String? = null
        var typeName: String? = null
        var typeMsg: String? = null
        var senderAccId: String? = null
    }
}
